### PR TITLE
L1 ht

### PR DESCRIPTION
**Pull request by taras-petushkov**
Lesson 1| see commit messages for more details

### CODE REVIEW

File: src/post.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                <div class="application-text__cta">Leggings rules the world!</div>
+            </div>
+            <div class="user-menu"></div>
+        </div>
+    </div>
+
+    <div class="content">
+        <div class="selected-post"></div>
+        <div class="related-posts__list"></div>
+    </div>
```
Comment (taras-petushkov) [inline src/post.html:25]: BEM is something I'm not good in.
I've just added class names similar to (as I saw it) posts.html class names:

```
.content -1> .selected-post -1> .post-preview && .selected-post__post -1> .post-preview__island -1> .post-preview__img-wrapper -1> .post-preview__img
.content -1> .selected-post -1> .post-preview && .selected-post__post -1> .post-preview__island -1> .post-preview__description
.content -1> .selected-post -1> .selected-post__comments -0..n> .selected-post__comment
.content -1> .related-posts__list -0..n> .post-preview && .related-posts__post -1> .post-preview__island -1> .post-preview__img-wrapper -1> .post-preview__img
.content -1> .related-posts__list -0..n> .post-preview && .related-posts__post -1> .post-preview__island -1> .post-preview__island -1> .post-preview__description
```

I think I'll read more about BEM and totally rebuild this page with some modularisation lesson.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "adv-fe",
+    "version": "1.0.0",
+    "description": "More details about the course can be found on http://frontend-science.com/advanced/.",
+    "author": "Petushkov Taras",
+    "dependencies": {
+        "express": "4.13.3"
+    },
+    "devDependencies": {
+        "bower": "1.7.7"
```
Comment (puzankov) [inline package.json]: Выполнить задание 3.2
 devDeps

File: package.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "adv-fe",
+    "version": "1.0.0",
+    "description": "More details about the course can be found on http://frontend-science.com/advanced/.",
+    "author": "Petushkov Taras",
```
Comment (puzankov) [inline package.json:5]: добавить npm start

File: .bowerrc [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+    "directory": "src/lib"
+}
```
Comment (puzankov) [inline .bowerrc]: update IDE config to add line in the end of file

File: src/post.html [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+<script type="text/x-handlebars-template" id="comment-template">
+    <div class="{{class}}">
+        <b>{{username}}</b>:{{text}}
+    </div>
+</script>
+
+<script type="text/x-handlebars-template" id="related-posts__list-template">
+    {{#each posts}}
+    {{> post-preview class="related-posts__post" }}
```
Comment (puzankov) [inline src/post.html]: use tab for child elems

File: .bowerrc (hunk 4 above)
Comment (taras-petushkov) [inline .bowerrc]: done

File: package.json (hunk 3 above)
Comment (taras-petushkov) [inline package.json:5]: done

File: package.json (hunk 2 above)
Comment (taras-petushkov) [inline package.json]: done

File: src/post.html (hunk 5 above)
Comment (taras-petushkov) [inline src/post.html]: done

**Comment by puzankov** [comment]
+1

**Pull request merged**